tests/php/Control/HTTPRequestBuilderTest.php

```diff
@@ -62,5 +61,14 @@ public function testExtractRequestHeadersBasicAuth()
             'PHP_AUTH_PW' => 'password',
         ];
         $this->assertEquals($headers, HTTPRequestBuilder::extractRequestHeaders($request));
+        
+        $request = [
+            'REDIRECT_HTTP_AUTHORIZATION' => 'Basic YWRtaW46cGFzc3dvcmQ=',
+        ];
+        $headers = [
+            'PHP_AUTH_USER' => 'admin',
+            'PHP_AUTH_PW' => 'password',
+        ];
+        $this->assertEquals($headers, HTTPRequestBuilder::extractRequestHeaders($request));
```

Comment:
can we add a case for when both `Authorization` and `REDIRECT_HTTP_AUTHORIZATION` are set?

I know it's not common, but sets expectations